crates/rethnet_evm_napi/index.d.ts

```diff
@@ -0,0 +1,617 @@
+/* tslint:disable */
```

Comment:
Similar to Slang, we should move auto-generated files to a dedicated folder, e.g. `generated`

Reply:
While I agree with this change, the current publish workflow assumes that the files are there and, as noted, it's pretty fragile, so I would rather not touch this and take care of it in https://github.com/NomicFoundation/edr/issues/198 if you don't mind.